.env.example

```
@@ -47,11 +47,18 @@ AXON_COLLECTION=cortex
 # --- Browser runtime knobs (optional) ---
 # [OPTIONAL] Primary WebDriver endpoint used for browser fallback.
-AXON_WEBDRIVER_URL=http://127.0.0.1:4444
+AXON_WEBDRIVER_URL=http://axon-webdriver:4444
 # [OPTIONAL] Compatibility fallback WebDriver endpoint when AXON_WEBDRIVER_URL is unset.
 WEBDRIVER_URL=
-# [OPTIONAL] Remote Chrome DevTools endpoint for browser crawling mode.
+# [OPTIONAL] Remote Chrome DevTools Protocol (CDP) endpoint for browser-mode crawling.
+# Used by axon engine to set website.with_chrome_connection() and by spider-rs directly
+# via CHROME_URL env var. Set both to the same host:port for full coverage.
+# Example: http://100.64.0.1:9222 (Tailscale-accessible Chrome DevTools instance)
 AXON_CHROME_REMOTE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AXON_CHROME_REMOTE_URL key should go before the AXON_WEBDRIVER_URL key

(UnorderedKey)